src/AppBundle/Entity/User.php

```diff
@@ -78,9 +78,9 @@ class User extends BaseUser
     /**
      * @var bool
      *
-     * @ORM\Column(type="boolean")
+     * @ORM\Column(type="boolean", nullable=true)
```

Comment:
Your default value for $wantToLearn property is false which contradicts the task that I am assigned.

Task given:
Remove pre-set of field in sign up form (step 2 in the sign up)